presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/PrestoConfigUtils.java

```diff
@@ -74,4 +73,123 @@ public static void deployConfigFiles(File configTar, Path configDir)
             throw new PrestoManagerException("Failed to deploy config files", untarResult);
         }
     }
+
+    public static void addConfigFiles(Path configDir)
+            throws PrestoManagerException
+    {
+        try {
+            if (isDirectory(DEFAULT_CONFIG_DIR)) {
+                copyDir(DEFAULT_CONFIG_DIR, configDir);
+                updateProperty(configDir.resolve("node.properties"), "node.id", randomUUID().toString());
+                addEnvSh(configDir);
+            }
+            else {
+                deleteDirectoryContents(configDir);
+                addWorkerConfig(configDir);
+            }
+        }
+        catch (IOException e) {
+            throw new PrestoManagerException("Failed to add config files", e);
+        }
+    }
+
+    private static void addEnvSh(Path configDir)
+            throws PrestoManagerException
+    {
+        Properties properties = new Properties();
+        properties.setProperty("JAVA8_HOME", System.getProperty("java.home"));
```

Comment:
we only care about setting java8_home when java8 is different from JAVA_HOME and not in a standard location, so this doesn't add anything.